src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/fs/FileSha1Step.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 CloudBees Inc.
```

Comment:
Did I miss a mail welcoming you to CloudBees :D if not then you can't claim copyright for anyone but yourself or the entity you are doing the work for ;)